presto-docs/src/main/sphinx/release/release-0.186.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.186
+=============
+
+General Changes
+---------------
+
+* Fix excessive GC overhead caused by map to map cast.
+* Fix implicit coercions for ``ROW`` types, allowing operations such as
+  comparisons between compatible types.
+* Fix issue that caused queries containing expensive functions, such as regular
+  expressions, to continue using CPU resources even after they were killed.
+* Fix :func:`json_parse` to not ignore trailing characters. Previously,
+  input such as ``[1,2]abc`` would successfully parse as ``[1,2]``.
+* Fix leak in running query counter for failed queries. The counter would
+  increment but never decrement for queries that failed before starting.
+* Reduce memory usage when building data of varchar or varbinary types.
+* Estimate memory usage for ``GROUP BY`` more precisely to avoid OOMs.
```

Comment:
maybe we shouldn't abbreviate `OOM`, not everyone may know what it is.

Reply:
This seems like a standard term of art. Searching Google, the Wikipedia "Out of memory" entry comes up second, and it's not just Java as you have the infamous "Linux OOM killer", etc. But I am fine with making it more explicit.

Reply:
Changed to "to avoid out of memory errors"